### PR TITLE
Added neovim resources

### PR DESCRIPTION
**Pull request by manvith12**
## What does this PR do?
Add resource

## For resources
Added neovim book , [learn neovim]( https://ofirgall.github.io/learn-nvim/)
resolves issue #11569 

### Why is this valuable (or not)?
adds books to neovim language

### How do we know it's really free?
its maintained by a github repo 

### For book lists, is it a book? For course lists, is it a course? etc.
yes its a book

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by Ajay-singh1 — CHANGES_REQUESTED** [review]
No need to create a new section add it under the IDE and Editors Section

**Review by manvith12 — COMMENTED** [review]
did the above cited suggestions

**Comment by manvith12** [comment]
cant find the linter error T-T

**Comment by manvith12** [comment]
lint left yayy

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1543,10 +1543,11 @@ Books on general-purpose programming that don't focus on a specific language are
 #### Emacs Lisp
 
 > :information_source: See also &#8230; [IDE and editors](free-programming-books-subjects.md#ide-and-editors)
-
 * [An Introduction to Programming in Emacs Lisp](https://www.gnu.org/software/emacs/manual/eintr.html)
 * [Elisp Programming](https://caiorss.github.io/Emacs-Elisp-Programming/Elisp_Programming.html)
 * [GNU Emacs Lisp Reference Manual](http://www.gnu.org/software/emacs/manual/elisp.html)
+* [GNU Emacs Lisp Reference Manual](http://www.gnu.org/software/emacs/manual/elisp.html)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1549]: you've added a duplicate

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1543,10 +1543,11 @@ Books on general-purpose programming that don't focus on a specific language are
...
+* [Learn Neovim](https://ofirgall.github.io/learn-nvim/) (HTML)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1550]: should go in the IDE and Editors section

**Review by eshellman — CHANGES_REQUESTED** [review]

**Comment by Ajay-singh1** [comment]
@manvith12 Kindly close this PR We have got a PR merged for #11569 by #11586.Thank you.

**Pull request closed without merging**